viewing_party/party.py

```diff
@@ -1,23 +1,112 @@
-# ------------- WAVE 1 --------------------
-
+# WAVE 1
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
+    if not title or not genre or not rating:
+        return None
+    movie["title"] = title
+    movie["genre"] = genre
+    movie["rating"] = rating
+    return movie

```

Comment:
Rather than creating an empty dictionary, then assigning the keys, we can use dictionary literal syntax to create the dictionary all at once.

```python
    movie_dict = { 'title': title, 'genre': genre, 'rating': rating }
    return movie_dict
```

It could even be done all on one line
```python
    return { 'title': title, 'genre': genre, 'rating': rating }
```